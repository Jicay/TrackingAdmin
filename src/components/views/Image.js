import React, {Component} from "react";

export default class Image extends Component {

    constructor(props) {
        super(props);
    }

    render() {
        const {image} = this.props;

        return (
            <div className='col align-content-center'>
                <img className="img-fluid" src={'https://live.staticflickr.com/' + image + '.jpg'}/>
            </div>
        )
    }


}