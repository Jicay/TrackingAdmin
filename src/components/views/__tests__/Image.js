import React from 'react';
import renderer from 'react-test-renderer';
import Image from "../Image";

it('renders correctly when home', () => {
    const tree = renderer
        .create(<Image image='skfjlsf/psdofispdofi'/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});