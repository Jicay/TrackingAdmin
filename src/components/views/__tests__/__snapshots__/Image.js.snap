// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when home 1`] = `
<div
  className="col align-content-center"
>
  <img
    className="img-fluid"
    src="https://live.staticflickr.com/skfjlsf/psdofispdofi.jpg"
  />
</div>
`;
